src/i_oalsound.c

```diff
@@ -735,6 +733,33 @@ void I_OAL_StopSound(int channel)
     alSourceStop(oal->sources[channel]);
 }
 
+void I_OAL_PauseSound(int channel)
+{
+    if (!oal)
+    {
+        return;
+    }
+
+    alSourcePause(oal->sources[channel]);
```

Comment:
A side effect to this is no more chainsaw "rev up" sound bug when pausing the game.